runtime/kusama/src/governance/mod.rs

```diff
@@ -47,6 +47,7 @@ impl pallet_conviction_voting::Config for Runtime {
 	type Currency = Balances;
 	type VoteLockingPeriod = VoteLockingPeriod;
 	type MaxVotes = ConstU32<512>;
+	// TODO: Should be `total_issuance` minus funds trapped in system pots (NIS, crowdloans, treasury)
 	type MaxTurnout = frame_support::traits::TotalIssuanceOf<Balances, Self::AccountId>;
```

Comment:
started looking into this, for crowdloans we probably need additional storage value, to store the total.